utils/apitpdata.go

```diff
@@ -1476,9 +1475,10 @@ type TPRate struct {
 
 type TPIntervalRate struct {
 	IntervalStart string
-	Value         float64 // RateValue
+	FixedFee      float64
 	Unit          string
 	Increment     string
+	RecurrentFee  float64 // RateValue
```

Comment:
Please configure them in the correct order everywhere, FixedFee, RecurrentFee, Unit, Increment.